CHANGELOG.md

```diff
@@ -3,6 +3,47 @@
 All notable changes to this project will be documented in this file.
 See [Conventional Commits](https://conventionalcommits.org) for commit guidelines.
 
+## 6.0.0-alpha.7
+
+_Nov 10, 2022_
+
+We'd like to offer a big thanks to the 5 contributors who made this release possible. Here are some highlights ✨:
+
+- 📚 Documentation improvements
+- 🐞 Bugfixes
```

Comment:
Can we add a bit more here? Here are new components from the pickers and a bunch of deprecations.

```suggestion
- ⏰ New `Time` and `DateTime` pickers
- ⚙️ Removed everything marked as `@deprecated`
- 📚 Documentation improvements
- 🐞 Bugfixes
```

Reply:
@flaviendelangle asked not to add the highlight for the new components as of now, we can add the deprecated thing though.